app/views/RoomView/Banner.js

```diff
@@ -22,7 +23,7 @@ const Banner = React.memo(
 						style={[styles.bannerContainer, { backgroundColor: themes[theme].bannerBackground }]}
 						testID='room-view-banner'
 						onPress={toggleModal}>
-						<Markdown msg={text} theme={theme} numberOfLines={1} style={[styles.bannerText]} preview />
+						<Preview msg={text} theme={theme} numberOfLines={1} style={[styles.bannerText]} />
```

Comment:
```suggestion
						<Preview msg={text} theme={theme} numberOfLines={1} style={styles.bannerText} />
```

Reply:
I understand why you are asking that, but actually here: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3558/files#diff-acf5acde58285628166dc989f5f1b1f4b7a4a928f0a46d2b0405045eb01aaf26R38

We are using the spread operator for every style that is coming, so I think we can continue with this array, just to maintain the pattern as other components, and to be isn't necessary to check if it is an array, WDYT?